docs/design/2022-07-03-block-storage-snapshot-based-backup-restore.md

```diff
@@ -0,0 +1,1494 @@
+# TiDB Cluster on Amazon EBS Backup & Restore
```

Comment:
move into `br/docs/`?

Reply:
I do not have an idea on it, I put previous the design in that folder, maybe one repo, one design folder? /cc @IANTHEREAL